configs/theta/config.20171031.tenYearTest.GMPAS-IAF.T62_oEC60to30v3wLI.60layer.theta

```diff
@@ -145,7 +136,7 @@ baseDirectory =  /projects/OceanClimate/mpas-analysis_data/ACMEv0_lowres/B1850C5
 polarPlot = False
 
 [streamfunctionMOC]
-## options related to plotting the streamfunction of the meridional overturning 
```

Comment:
wow, we do have a ton of config files..

Reply:
Yes, true.  But they are all for different resolutions, or runs with and without ice-shelves or different beta versions.  And obviously for many machines.  So we really do need them.

Reply:
no no, I agree. I had just not noticed they have become so many :)